### DISCUSSION TITLE
为什么比特币可以防篡改 · Why's THE Design? · /whys-the-design-bitcoin-database

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#590] [Announcements]
https://draveness.me/whys-the-design-bitcoin-database 

 如果我们对比特币等区块链技术稍有了解，就会发现它是一个设计巧妙的分布式数据库。作为在公网运行的分布式数据库，比特币和其它区块链网络都会面对网络中的恶意节点的攻击。因为比特币需要面对复杂的网络环境以及不可靠的节点，所以它在设计和实现上也做出了应对，我们可以看看它是如何组合现有的技术防止恶意节点对交易和账户数据进行篡改的。

**Comment by wujunze** [comment]
比特币用经济杠杆 防止作弊 篡改   😆

**Comment by lycheng** [comment]
> MySQL 等数据库以行为单位存储数据，而比特币这个分布式数据库中存储的基本单位是区块，区块通过哈希指针连接就会构成一颗树

这里应该是一棵树吧？

**Comment by draveness** [comment]
> > MySQL 等数据库以行为单位存储数据，而比特币这个分布式数据库中存储的基本单位是区块，区块通过哈希指针连接就会构成一颗树
> 
> 这里应该是一棵树吧？

确实打错了，我改下

**Comment by yeahjjyy** [comment]
"而使用公钥解密的数据也只能用私钥解密"   这里的公钥解密应该是公钥加密吧

---

*2020-10-22 UPDATES: 已修复*

```diff
-公钥解密
+公钥加密
```

**Comment by 2250749979** [comment]
2021年考古  比特币突破6万美元  量子计算机可以破解公钥吗？
